saphanabootstrap-formula.spec

```diff
@@ -36,6 +36,7 @@ Suggests:       prometheus-hanadb_exporter >= 0.7.0
 %define fname hana
 %define fdir  %{_datadir}/salt-formulas
 %define ftemplates templates
+%define fmacros macros
```

Comment:
I don't think we need this change. All the content from `hana` folder should me already moved properly

Reply:
thanks I will check it